docs/deployments/changing-domains.mdx

```diff
@@ -1,5 +1,81 @@
 ---
-sanity_slug: /docs/deployments/change-production-domain
+title: Change domain or subdomain
+description: Learn how to change your production instance's domain or subdomain on Clerk.
 ---
 
-# This is a stub
+# Change domain or subdomain
+
+Learn how to change your production instance's domain or subdomain on Clerk.
+
+## Change domain
+
+Production domains can be changed in the [Clerk Dashboard](https://dashboard.clerk.com/) or via our [backend API](https://clerk.com/docs/reference/backend-api). Once you make the change to your domain, you will need to update the following:
+
+- Update the publishable key
```

Comment:
This should be moved to its own section below the section for the new SSL Certs. The customer can't update this first -- the new key is generated after the DNS changes. It is just the domain base64 encoded.

This is the step that causes most of the support request relating to changing domains. Making it super clear with its own section can only help. 

Maybe something like this (I used a code block to the CallOut tags would show): 

## Update Your Publishable Key

After changing your domain, a new Publishable Key was automatically generated for your application. You will need to update your production instance with this new key and redeploy your application. 

```
<Callout>
Failing to update your Publishable will result in Clerk failing to load. 
</Callout>
```